website/source/api/jobs.html.md

```diff
@@ -1380,11 +1382,30 @@ The table below shows this endpoint's support for
 - `:job_id` `(string: <required>)` - Specifies the ID of the job (as specified in
   the job file during submission). This is specified as part of the path.
 
+- `JobID` `(string: <required>)` - Specify the ID of the job in the JSON payload
+
+- `EvalOptions` `(<optional>)` - Specify additional options to be used during the forced evaluation.
+    - `ForceReschedule` `(bool: false)` - If set, any failed allocations of the job are rescheduled
```

Comment:
Remove any. For me it gives the connotation that all failed evals will be rescheduler but really we mean any failed alloc that doesn't already have a replacement